reinforcement_learning/rl_cartpole_coach/rl_cartpole_coach_gymEnv.ipynb

```diff
@@ -39,7 +39,18 @@
    "cell_type": "code",
```

Comment:
<p>We don't need this anymore, because it looks like the Sagemaker team has bumped up the default version of the SDK to 2.16.1.</p><p>Refer: https://github.com/aws/amazon-sagemaker-examples/pull/1697#issuecomment-723302690</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1720/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>